promscale/installation/helm.md

```diff
@@ -135,8 +118,10 @@ When you have installed the TimescaleDB and Promscale Helm charts, you can insta
     ```bash
     helm repo update
     ```
-1.  Configure the remote-read and write endpoints in Prometheus server configuration. As we are using 
-    Prometheus helm chart configure this values in `values.yaml` file as:
+1.  Configure the remote-read and write endpoints in the Prometheus server
+    configuration, in `values.yaml` configuration file. You can download the
+    `values.yaml` file from [our repository][prometheus-values-yaml]. Open the
+    file, locate the `server` section and configure the file like this:
```

Comment:
I also don't think that it makes sense to tell the user to download the prometheus `values.yaml`. There's too much confusing stuff in there.

Reply:
Also: if they follow these instructions, they will end up with two files which should be called `values.yaml` (the one for promscale, and now one for prometheus). I think that we should give a more specific name.

Reply:
We removed the Prometheus instructions so no longer a problem